src/debug/telemetry/telemetry2_slot.c

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
this seems to be a Zephyr licensing term, is this what we want for something in sof/ ?

Reply:
Oops, first wrote everything on the Zephyr side, and when I moved it sof side, the license was copied over. Will fix.